src/components/CardList/CardList.js

```diff
@@ -3,7 +3,7 @@ import Card from '../Card/Card';
 
 function CardList({ cards, type }) {
   return (
-    <ul className={`card-list card-list_type_${type}`}>
+    <ul className={`card-list card-list_type_${type} ${cards.length % 4 === 0 && 'card-list_spacing_space-between'}`}>
```

Comment:
This is a bit of a hack.  Having spacing-between set at 1280 will match the layout on figma precisely, but it does not look good (in my mind) when there are 5 or 6 cards (left justified).